templates/taxi/driver_list.html

```diff
@@ -1,30 +1,35 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
-    </h1>
+  <h1>Driver List
```

Comment:
Code Quality Issue: It's a good practice to close your HTML tags. The heading tag `<h1>` opened here is not closed. Consider closing it with `</h1>` for better HTML structure and readability.